x-pack/plugins/upgrade_assistant/index.ts

```diff
@@ -15,6 +15,12 @@ export function upgradeAssistant(kibana: any) {
     uiExports: {
       managementSections: ['plugins/upgrade_assistant'],
       styleSheetPaths: resolve(__dirname, 'public/index.scss'),
+      mappings: require('./mappings.json'),
+      savedObjectSchemas: {
+        'upgrade-assistant-reindex-operation': {
+          isNamespaceAgnostic: true,
```

Comment:
What is `isNamespaceAgnostic` for?

Reply:
This is to make this object type not space-specific.